docs/pages/pmd/userdocs/cli_reference.md

```diff
@@ -197,29 +207,34 @@ to be "debug".
 The language is determined automatically by PMD from the file extensions. Some languages such as "Java"
 however support multiple versions. The default version will be used, which is usually the latest supported
 version. If you want to use an older version, so that e.g. rules, that suggest usage of language features,
-that are not available yet, won't be executed, you need to specify a specific version via the `-language`
-and `-version` parameter.
+that are not available yet, won't be executed, you need to specify a specific version via the `--use-version`
+parameter.
 
 These parameters are irrelevant for languages that don't support different versions.
 
 Example:
 
 ``` shell
-./run.sh pmd -d src/main/java -f text -R rulesets/java/quickstart.xml -language java -version 8
+./pmd analyze -d src/main/java -f text -R rulesets/java/quickstart.xml --use-version java-1.8
 ```
 
 *   [apex](pmd_rules_apex.html) (Salesforce Apex)
+*   [ecmascript](pmd_rules_ecmascript.html) (JavaScript)
+*   [html](pmd_rules_html.html)
```

Comment:
interestingly, some of these missing languages are even supported in PMD 6, but we don't mention them on this list

Reply:
This is done for PMD6 with #4144